app/src/main/java/com/asfoundation/wallet/wallet_reward/RewardFragment.kt

```diff
@@ -105,11 +99,12 @@ class RewardFragment : BasePageViewFragment(), SingleStateFragment<RewardState,
         .padding(padding),
     ) {
       DummyCard()
```

Comment:
could you remove this dummy?

Reply:
This DymmyCard is for the last card in Rewards, Gamification